src/rules/memberOrderingRule.ts

```diff
@@ -57,19 +57,28 @@ function toString(modifiers: IModifiers): string {
 export class MemberOrderingWalker extends Lint.RuleWalker {
     private previous: IModifiers;
 
-    constructor(sourceFile: ts.SourceFile, options: Lint.IOptions) {
-        super(sourceFile, options);
-    }
-
-    public visitClassDeclaration(node: ts.ClassDeclaration): void {
+    private resetPrevious(): void {
```

Comment:
please move this method towards the end of the class (after all public methods). also i think "resetModifiers" or "resetPreviousModifiers" is a better name.